code/modules/RCD/schematics/pipe.dm

```diff
@@ -382,8 +382,8 @@
 /datum/rcd_schematic/pipe/disposal/attack(var/atom/A, var/mob/user)
 	to_chat(user, "Building Pipes ...")
 	playsound(get_turf(user), 'sound/machines/click.ogg', 50, 1)
-	if(!do_after(user, A, 20))
-		return 1
+//	if(!do_after(user, A, 1))
+//		return 1
```

Comment:
You need a return 1 on attack so it registers as a succesful attack, so some other sanity is ran IIRC